### ISSUE TITLE
Parser Error: Class "PhantomCamera2D" hides a global script class.

### ISSUE THREAD

**Issue by luiscesjr** [ramokz/phantom-camera#272]
### Issue description

I just came back to my office from the weekend, and I got the update notice as soon as I opened Godot. After updating, I intended to do some playtesting but I was greeted with this error "Parser Error: Class "PhantomCamera2D" hides a global script class." As also shown in the image below.

![image](https://github.com/ramokz/phantom-camera/assets/7636427/c410db77-0b66-4b31-833f-c701c9e39ddd)

I simply updated the add on from the auto updater, maybe I missed something?

Tried deleting .godot folder, delisting and relisting my project, reimporting all, but no dice.

And also tried to manually download the plugin again, also no luck there.

As a bonus I also get these warnings now:

![image](https://github.com/ramokz/phantom-camera/assets/7636427/6f67da76-a265-4604-879a-6663edb1ca74)

### Steps to reproduce

Update your add-on.

**Comment by LenanYamao** [comment]
I had the same issue, i had to delete phantom_camera_2d from my scene and add it back. If you look at your phantom camera it will probably look like this

![image](https://github.com/ramokz/phantom-camera/assets/43767142/5fb5c250-612b-4dea-a272-62a024fdf968)

You can also remove the script from your node and add it back, but you still have to reconfigure your camera

**Comment by luiscesjr** [comment]
> I had the same issue, i had to delete phantom_camera_2d from my scene and add it back. If you look at your phantom camera it will probably look like this

I can confirm that "This is the way". Worked for me, thanks!

**Comment by ramokz** [comment]
The same issue was reported yesterday in #263.

Tldr; Godot is weird with class name changes. The issue was solved for the poster by reopening the Godot project several times.

It's a seemingly random occurring bug.

**Comment by luiscesjr** [comment]
> The same issue was reported yesterday in #263.
> 
> Tldr; Godot is weird with class name changes. The issue was solved for the poster by reopening the Godot project several times.
> 
> It's a seemingly random occurring bug.

I'm sorry for double posting the issue, I looked through the recent open bugs and forgot to check the closed ones. And yes, it's quite random, occurred to my own created libraries not long ago.

**Comment by ramokz** [comment]
No problem! I do sometimes find it difficult to locate issues on GitHub as well, particularly when the search defaults to not show closed issues…

Closing this as it sounds like it's resolved now.
